tests/foreman/ui/test_setting.py

```diff
@@ -59,9 +59,18 @@ def valid_maxtrend_timeout_values():
 def valid_urls():
     """Returns a list of valid urls"""
     return [
-        gen_url(subdomain=gen_string('alpha')),
-        gen_url(subdomain=gen_string('alphanumeric')),
-        gen_url(subdomain=gen_string('numeric')),
+        gen_url(
+            scheme=choice(('http', 'https')),
```

Comment:
Good example of random failure :)
Without specifying any protocol, there's 33% chance that 'ftp' will be selected. `valid_urls()` is used in Satellite URL setting tests, and for obvious reasons validation won't allow to change sat URL to url with ftp protocol.